data.sql

```diff
@@ -82,4 +82,9 @@ INSERT INTO visits (animal_id, vet_id, visit_date)
       ((SELECT id FROM animals WHERE name = 'Blossom'), (SELECT id FROM vets WHERE name = 'Stephanie Mendez'), '2020-05-24'),
       ((SELECT id FROM animals WHERE name = 'Blossom'), (SELECT id FROM vets WHERE name = 'William Tatcher'), '2021-01-11');
 
+/* Add 3.594.280 visits considering 10 animals, 4 vets, and it will use around 87.000 timestamps */
 
+INSERT INTO visits (animal_id, vet_id, visit_date) SELECT * FROM (SELECT id FROM animals) animal_ids, (SELECT id FROM vets) vets_ids, generate_series('1980-01-01'::timestamp, '2021-01-01', '4 hours') visit_timestamp;
+
```

Comment:
I noticed that in one of your **before** screenshots your execution time is significantly less than 1sec (<1000ms)

![image](https://user-images.githubusercontent.com/38088818/192378088-fb9b704b-8765-465d-bb75-c9ab64be0c7c.png)

- Kindly try to run the populate database query a few times more as described in the [project requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_performance_audit.md#:~:text=4%20Depening%20on,bigger%20than%201000ms.). Depending on your machine speed, we might need to run this query multiple times to reach at least 1sec (100ms).

NB: You might need to update your screenshots after successfully completing this task.

Reply:
@KDlamini Thank you for pointing this out! :+1: Here I inserted more records and ran the performance audit again :heavy_check_mark: 
### Performance audit results (before and after) :camera_flash: 
![performance-audit-2](https://user-images.githubusercontent.com/92395484/192383958-b49fdc73-9a14-4ede-a483-e5ffa4034685.png)

Reply:
@karlavdelgadof Great job 👍🏾👏🏾